README.md

```diff
@@ -8,32 +11,94 @@
+- **Customizable**: Extensive configuration options to tailor the faction experience to your server's needs.
+- **Minecraft Integration**: Unique features downloadable directly through Minecraft for seamless user experience.
+- **Regular Updates**: Continual improvements and new features to keep your faction gameplay fresh and exciting.
+
+## Installation
+
+1. Download the latest version of ImprovedFactions from the [Spigot page](https://www.spigotmc.org/resources/improved-factions.95617) or [GitHub releases](https://github.com/ToberoCat/ImprovedFactions_new/releases/).
+2. Place the downloaded `.jar` file in your server's `plugins` folder.
+3. Restart your server.
+4. The plugin will generate default configuration files, which you can then customize.
+
+## Usage
+
+- `/f help` - View all available faction commands
+- `/f create <name>` - Create a new faction
+- `/f join <faction>` - Join an existing faction
+- `/f claim` - Claim land for your faction
+
+For a full list of commands and their usage, please refer to our [documentation](#documentation).
+
+## Configuration
+
+The main configuration file is located at `plugins/ImprovedFactions/config.yml`. Here you can adjust various settings such as:
+
+- Faction creation costs
+- Claim limits
+- Power systems
+- And much more!
+
+Detailed configuration instructions can be found in our [wiki](https://github.com/ToberoCat/ImprovedFactions_new/wiki).
+
+## Documentation
 
 [![view - Documentation](https://img.shields.io/badge/view-Documentation-blue?style=for-the-badge)](https://github.com/ToberoCat/ImprovedFactions_new/wiki "Go to project documentation")
 
-</div>
+Our comprehensive documentation covers everything from basic setup to advanced features. Check out our [GitHub Wiki](https://github.com/ToberoCat/ImprovedFactions_new/wiki) for detailed guides and information.
 
-## About
+## Contributing
+
+We welcome contributions from the community! If you'd like to contribute, please:
+
+1. Fork the repository
+2. Create a new branch for your feature
+3. Commit your changes
+4. Push to the branch
+5. Create a new Pull Request
 
-Improved factions is a lightweight and performant minecraft faction plugin. It supports features downloadable from
-Minecraft directly for more user friendlienes
+For more information, see our [CONTRIBUTING.md](CONTRIBUTING.md) file.
 
-## Contributors
+## Support
 
-Check [CONTRIBUTORS.md](CONTRIBUTORS.md) to see a list of all people who contributed to this project
+- For bug reports and feature requests, please use our [GitHub Issues](https://github.com/ToberoCat/ImprovedFactions_new/issues).
+- Join our [Discord server](https://discord.gg/VmSbFNZejz) for community support and discussions.
 
-## Others
+## License
 
-### Bstat
-https://bstats.org/plugin/bukkit/ImprovedFactions/14810
+ImprovedFactions is released under the [MIT License](LICENSE).
+
+---
+
+<div align="center">
+
+Made with ❤️ by [ToberoCat](https://github.com/ToberoCat) and [contributors](CONTRIBUTORS.md).
+
```

Comment:
Move `Check [CONTRIBUTORS.md](CONTRIBUTORS.md) to see a list of all people who contributed to this project` to the footer, I think it'd look quite nice while keeping the original message 👍